src/ImageSharp/Diagnostics/MemoryDiagnostics.cs

```diff
@@ -69,10 +68,16 @@ internal static void RaiseUndisposedMemoryResource(string allocationStackTrace)
             }
 
             // Schedule on the ThreadPool, to avoid user callback messing up the finalizer thread.
+#if NETSTANDARD2_1 || NETCOREAPP2_1_OR_GREATER
```

Comment:
https://apisof.net/catalog/737fc3e3-23d4-dfab-294f-8c59c58c5852

Reply:
Damn, thanks!

Reply:
No worries. Thought I'd better wait until you'd seen this before merging. I'll let you do the honors when you're happy.